packages/select/src/index.mts

```diff
@@ -76,7 +77,17 @@ export default createPrompt(
       return { first, last };
     }, [items]);
 
-    const [active, setActive] = useState(bounds.first);
+    const defaultItemIndex = useMemo(() => {
+      if (!_default) return -1;
```

Comment:
```suggestion
      if (!'default' in config) return -1;
```

Thinking we might want to be more liberal with the value we allow to provide 🤔 (like allowing falsy values, null or undefined)